medicines/api/infrastructure/production/.env.example

```diff
@@ -0,0 +1,3 @@
+export AZURE_SEARCH_INDEX=
```

Comment:
`.env` inconsistency here: `export` is not included in the other examples

Reply:
Good catch